src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -152,14 +153,9 @@ private static bool AudienceIsValid(IEnumerable<string> audiences, TokenValidati
         private static bool AudiencesMatch(TokenValidationParameters validationParameters, string tokenAudience, string validAudience)
         {
             if (validAudience.Length == tokenAudience.Length)
-            {
-                if (string.Equals(validAudience, tokenAudience))
-                    return true;
-            }
-            else if (validationParameters.IgnoreTrailingSlashWhenValidatingAudience && AudiencesMatchIgnoringTrailingSlash(tokenAudience, validAudience))
-                return true;
-
-            return false;
+                return string.Equals(validAudience, tokenAudience);
+            else
```

Comment:
nit:

When you have an else at the end of an if/elseif/.../else chain, you can drop the else and move that up a level. It makes it slightly more clear that it is the default case.

Reply:
changed